src/client/Microsoft.Identity.Client/Internal/Requests/ClientCredentialRequest.cs

```diff
@@ -86,6 +83,24 @@ protected override async Task<AuthenticationResult> ExecuteAsync(CancellationTok
             }
         }
 
+        private void validateRequest()
```

Comment:
Please use PascalCasing, i.e. `ValidateRequest` for all methods, private and public.